### ISSUE TITLE
ListView page number should be displayed as get parameters

### ISSUE THREAD

**Issue by akovardakov** [best-doctor/ke#32]
Otherwise location is not accurate

**Comment by mcproger** [comment]
True. We have already done this with filters, the same is needed for pagination parameters. You can use as example [this](https://github.com/best-doctor/ke/blob/master/src/components/Table/filters.tsx#L30) and [this](https://github.com/best-doctor/ke/blob/master/src/ResourceComposer/RenderList.tsx#L40)
